policy/release/tasks.rego

```diff
@@ -197,11 +197,40 @@ current_required_tasks := task_data if {
 
 ## get the required task data for a pipeline with a label
 required_pipeline_task_data := task_data if {
-	some att in lib.pipelinerun_attestations
+	some att in _pipelinerun
 	count(tkn.tasks(att)) > 0
 	task_data := tkn.required_task_list(att)
 }
 
-_status(task) := status if {
+_status(task) = status if {
+	not task.status.conditions
 	status := task.status
+} else = status if {
+	some condition in task.status.conditions
```

Comment:
The status is actually a list of conditions?!

Reply:
Here's what I noticed in the taskRun. Maybe I need to check `reason` also?
```json
"status": {
    "conditions": [
      {
        "type": "Succeeded",
        "status": "True",
        "lastTransitionTime": "2023-09-15T19:17:45Z",
        "reason": "Succeeded",
        "message": "All Steps have completed executing"
      }
    ],
```

Reply:
Wait, this is in the SLSA Provenance right? Not the resource from the cluster?

Also, it would be really good if you could share an example SLSA Provenance v1.0 generated by Chains as a gist(?) in this PR :pray:

Reply:
And if it's a list, can there be multiple entries? idk, I'm going to look into this more.

Reply:
I think it should be ok if it this rule produces multiple results. It just means that whatever calls `_status` may also produce multiple results.

Reply:
maybe it could have been simpler, but I ended up with this. The slsav0.2 status is not an array, but the v1.0 is and the way it was written it only returned one value no matter what.
https://github.com/enterprise-contract/ec-policies/pull/709/commits/e668fc09bf2f7f10da036d48694e6a85c76a108a